File: samples/client/3_1_0_unit_test/java/src/main/java/unit_test_api/components/schemas/TypeAsArrayWithOneItem.java
package unit_test_api.components.schemas;
import org.checkerframework.checker.nullness.qual.Nullable;
import unit_test_api.schemas.StringJsonSchema;

public class TypeAsArrayWithOneItem extends StringJsonSchema {
    // nest classes so all schemas and input/output classes can be public
    
    
    public static class TypeAsArrayWithOneItem1 extends StringJsonSchema.StringJsonSchema1 {
        private static @Nullable TypeAsArrayWithOneItem1 instance = null;
        public static TypeAsArrayWithOneItem1 getInstance() {
            if (instance == null) {
                instance = new TypeAsArrayWithOneItem1();
            }
            return instance;
        }
    }

}

File: samples/client/3_1_0_unit_test/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/schemas/TypeAsArrayWithOneItem.kt
package org.openapijsonschematools.client.components.schemas
import org.openapijsonschematools.client.schemas.StringJsonSchema

class TypeAsArrayWithOneItem: StringJsonSchema() {
    // nest classes so all schemas and input/output classes can be public
    
    
    class TypeAsArrayWithOneItem1 : StringJsonSchema.StringJsonSchema1() {
        companion object {
            @Volatile
            private var instance: TypeAsArrayWithOneItem1? = null
    
            fun getInstance() =
                instance ?: synchronized(this) {
                    instance ?: TypeAsArrayWithOneItem1().also { instance = it }
                }
        }
    }

}
